Crypto-Currency/Crypto-Currency/Model/Networking/RepositoryAPI.swift

```diff
@@ -21,44 +21,67 @@ class RepositoryAPI {
     }
     
     func getMore(offset: String, completion: @escaping ([Coin]?, Error?) -> Void) {
-        APIService.shared.request(urlString: "\(EndPoint.offset.rawValue)\(offset)",
-                                  expecting: ResponseRanking<DataCoin>.self) { result in
+        APIService.shared.request(
+            urlString: Network.shared.getOffsetURL(offset: offset),
+            expecting: ResponseRanking<DataCoin>.self) { result in
```

Comment:
APIService.shared.request(urlString: "\(EndPoint.offset.rawValue)\(offset)",
                                  expecting: ResponseRanking<DataCoin>.self) { result in